### PR TITLE
[DOCS] NAPALM Installation Documentation Broken Link Correction

### LINKED ISSUE #56501: napalm page has dead hyperlink
Relevant salt docs page:
- https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.napalm_netacl.html

Relevant salt docs file and error:
```
writing output... [ 26%] ref/modules/all/salt.modules.napalm_netacl

(line   22) broken    https://napalm.readthedocs.io/en/latest/installation.html - 404 Client Error: Not Found for url: https://napalm.readthedocs.io/en/latest/installation.html
```
Comments
- manually verified

### LINKED ISSUE #56511: Napalm yang state module docs has deadlink
Relevant salt docs page:
- https://docs.saltstack.com/en/latest/ref/states/all/salt.states.net_napalm_yang.html

Relevant salt docs file and error:
```
writing output... [ 65%] ref/states/all/salt.states.net_napalm_yang

(line   15) broken    https://napalm.readthedocs.io/en/latest/installation.html - 404 Client Error: Not Found for url: https://napalm.readthedocs.io/en/latest/installation.html
```
Comments
- manually verified

### LINKED ISSUE #56512: netacl states docs page has dead hyperlink
Relevant salt docs page:
- https://docs.saltstack.com/en/master/ref/states/all/salt.states.netacl.html

Relevant salt docs file and error:
```
writing output... [ 65%] ref/states/all/salt.states.netacl

(line   27) broken    https://napalm.readthedocs.io/en/latest/installation.html - 404 Client Error: Not Found for url: https://napalm.readthedocs.io/en/latest/installation.html

```
Comments
- manually verified

### PR DESCRIPTION
**Pull request by ggiesen**
### What does this PR do?
Corrects broken link to NAPALM installation documents

Staged in two commits per issue, one for URL update, one for formatting changes from pre-commit

### What issues does this PR fix or reference?
Fixes: #56501
Fixes: #56511
Fixes: #56512
Fixes: saltstack/docs-hub#31
Fixes: saltstack/docs-hub#22

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [x] Docs

### Commits signed with GPG?
No

### CODE REVIEW

File: salt/modules/napalm_netacl.py [hunk 1]
```diff
@@ -26,9 +25,8 @@
 Please check Installation_ for complete details.
 
 .. _NAPALM: https://napalm.readthedocs.io
-.. _Installation: https://napalm.readthedocs.io/en/latest/installation.html
+.. _Installation: https://napalm.readthedocs.io/en/latest/installation/index.html
```
Comment (ggiesen) [inline salt/modules/napalm_netacl.py:28]: These are the actual changes. All other changes are due to pre-commit

File: salt/states/netacl.py [hunk 2]
```diff
@@ -31,7 +31,7 @@
 Please check Installation_ for complete details.
 
 .. _NAPALM: https://napalm.readthedocs.io
-.. _Installation: https://napalm.readthedocs.io/en/latest/installation.html
+.. _Installation: https://napalm.readthedocs.io/en/latest/installation/index.html
```
Comment (ggiesen) [inline salt/states/netacl.py:33]: These are the actual changes. All other changes are due to pre-commit

File: salt/states/net_napalm_yang.py [hunk 3]
```diff
@@ -19,7 +19,7 @@
 Please check Installation_ for complete details.
 
 .. _NAPALM: https://napalm.readthedocs.io
-.. _Installation: https://napalm.readthedocs.io/en/latest/installation.html
+.. _Installation: https://napalm.readthedocs.io/en/latest/installation/index.html
```
Comment (ggiesen) [inline salt/states/net_napalm_yang.py:21]: These are the actual changes. All other changes are due to pre-commit

**Comment by ggiesen** [comment]
re-run pr-freebsd122-py3-pytest

**Review by mirceaulinic — APPROVED** [review]
LGTM

**Review by ScriptAutomate — APPROVED** [review]

**Review by Ch3LL — APPROVED** [review]

**Pull request merged**